tests/end2end/conftest.py

```diff
@@ -0,0 +1,42 @@
+"""
+Module for global PyTest configuration and fixtures
+
+"""
+
+import re
+
+import pytest
+import psutil
+
+
+from helpers import kill_process
+
+_PORT = 50052
+
+@pytest.fixture(scope='session')
+def port():
+    return str(_PORT + 1)
+
+
+@pytest.fixture(scope='module', autouse=True)
+def post_session(request):
+    """This method makes sure that the environment is clean to execute another test"""
+    print(f"\n #########  Running test {request.node}:{request.node.name} ---------------------------")
+
+    yield
+
+    print("#### Checking remaining processes if there is any not killed after the tests")
+    for process in psutil.process_iter():
+        try:
+            cmdline = process.cmdline()
+        except psutil.Error:
+            continue
+        else:
+            if any([re.search(r'^end2end_.*\.ini$', cmd) for cmd in cmdline]):
```

Comment:
Just a warning flag, I'm not sure that I understand the context 100%, but this is just to say that the regex is quite "generic" and could potentially catch unwanted processes, since `end2end` is quite a common name. 

Maybe I'm not understanding this correctly and the search is really in a very small subset of processes, but still, just wanted to signal this and make sure we have thought about it and confirmed that it's ok.

Reply:
Hi @sharkovsky, 

Thank you for the warning. I was counting on `.ini` extension but still you are right that it is possible to catch other processes. I am going to add remaining part of the prefix `^end2end_component`.